docs/relational-databases/performance/best-practice-with-the-query-store.md

```diff
@@ -450,8 +451,29 @@ Use the following sample extended event session to capture non-parameterized que
   ADD TARGET package0.event_file(SET filename=N'QueryStore',max_file_size=(100))
   WITH (MAX_MEMORY=4096 KB,EVENT_RETENTION_MODE=ALLOW_SINGLE_EVENT_LOSS,MAX_DISPATCH_LATENCY=30 SECONDS,MAX_EVENT_SIZE=0 KB,MEMORY_PARTITION_MODE=NONE,TRACK_CAUSALITY=OFF,STARTUP_STATE=OFF)
 ```
+The above extended event capture the event query_store_db_diagnostics, with this data you will find out the plan count in the Query Store and also many other stats as well.
```

Comment:
@PiJoCoder, I added 2 more queries for identifying the Parameterized Queries and Non-Parameterized Queries , As per the Jay , this extended event is help in case high volume of Non-Parameterized Queries it means high volume of plan count. So this extended event will indicate where is the root cause of performance issue.
![QueryStore](https://user-images.githubusercontent.com/62662636/162020389-cfd1e3c4-8922-4401-9c36-f67c1839b8aa.JPG)

Reply:
I just tried the same at my local machine

Reply:
Yes, the added queries are great! Thanks for adding those - good learning experience for me. What William was asking is to tell the customer if they use the Xevent, what columns to look at - be explicit about it. That's why I am proposing this addition right here after this line.

```suggestion
The above extended event capture the event query_store_db_diagnostics, with this data you will find out the plan count in the Query Store and also many other stats as well. Look at the plan_count, query_count, max_stmt_hash_map_size_kb, max_size_mb columns in the event to understand the amount of memory used and number of plans that are tracked by Query Store.
```

Reply:
Thanks @PiJoCoder  and @WilliamDAssafMSFT  for helping here. It will be a great learning for me as well. I have added the suggestions.

Reply:
I've made a couple commits to improve, please review. I think we're close! Any more direct info we can point to in the results of the `query_store_db_diagnostics` output?

Reply:
@WilliamDAssafMSFT, I don't have handy to add more with respect to query_store_db_diagnostics output. Please merge it for now, if something comes, we will raise a new pull request.
Thank for your time and support on this, I appreciate it a lot.

Reply:
OK, made a few more style changes and edits to the code samples. I think we're ready to go now.